StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -76,7 +76,7 @@ private void HandleTypeDeclaration(SyntaxNodeAnalysisContext context)
             {
                 if (!XmlCommentHelper.HasDocumentation(declaration))
                 {
-                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation()));
+                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation(), ArrayEx.Empty<object>()));
```

Comment:
But I don't know what ver of mscorlib used there
http://tryroslyn.azurewebsites.net/#K4Zwlgdg5gBAygTxAFwKYFsDcAoADsAIwBswBjGUogQxBBgGEYBvbGNmfYsmANwHswAExgARABQBKZq3ayAspJyyAvjLZqOhEuX5CYC3FQBOVdHT4EAVqlLIA2gF0YfKSxXZlQAA